### ISSUE TITLE
link to view.add

### ISSUE THREAD

**Issue by ben-github** [dpgaspar/Flask-AppBuilder#77]
Please pardon the double-post but I commented on a closed issue which wasn't great, so I'm reposting on a new issue. I'm trying to add a link in the menu to view's add page. Per your suggestion in #76, I tried 

```
appbuilder.add_link("Test Add Item", href=url_for(TestView.add), icon="fa-folder-open-o", category="Testing")
```

but, when I try to use it get the error:
```
 File "/usr/lib/python2.7/dist-packages/flask/helpers.py", line 258, in url_for
    raise RuntimeError('Attempted to generate a URL without the '
RuntimeError: Attempted to generate a URL without the application context being pushed. This has to be executed when application context is available.
```

Am I missing something? I have
```
class TestView(ModelView):
    datamodel=SQLAModel(Test)

    
appbuilder.add_view(TestView, "Test", icon="fa-folder-open-o", category="Testing")
appbuilder.add_link("Test Add Item", href=url_for(TestView.add), icon="fa-folder-open-o", category="Testing")
```
    
Thanks

**Comment by dpgaspar** [comment]
Your right, my mistake. there is no request context when appbuilder.add_link is used. I will get a solutions for this on add_link.

**Comment by dpgaspar** [comment]
This is solved on 1.3.0 with add_link

You can use it like this:

    class TestView(ModelView):
        datamodel=SQLAModel(Test)


    appbuilder.add_view(TestView, "Test", icon="fa-folder-open-o", category="Testing")
    appbuilder.add_link("Test Add Item", "TestView.add", icon="fa-folder-open-o", category="Testing")
